static/js/global_init.js

```diff
@@ -5,10 +5,14 @@ global.cancelAnimationFrame = () => {}
 global.requestAnimationFrame = () => {}
 
 const _createSettings = () => ({
-  micromasters_external_login_url: "http://fake.micromasters.url/discussions",
-  micromasters_base_url:           "http://fake.micromasters.url/",
-  session_url:                     "http://fake.session.url",
-  max_comment_depth:               6
+  max_comment_depth:  6,
+  authenticated_site: {
+    title:       "MicroMasters",
+    login_url:   "http://fake.micromasters.url/discussions",
```

Comment:
I think this should be `auth_url` since that's what's output in `views.py` in `js_settings`

Reply:
I swear I refactored that.

Reply:
@noisecapella actually, it is `login_url` in views.py: https://github.com/mitodl/open-discussions/pull/492/files#diff-125966fe4af6f53bae3569a94811ca46R52

Reply:
Oh right